install.sh

```diff
@@ -44,10 +44,10 @@ function fancy_message() {
 	local MESSAGE="${2}"
 
 	case ${MESSAGE_TYPE} in
-		info) echo -e "[${GREEN}+${NC}] INFO: ${MESSAGE}" ;;
-		warn) echo -e "[${YELLOW}*${NC}] WARNING: ${MESSAGE}" ;;
-		error) echo -e "[${RED}!${NC}] ERROR: ${MESSAGE}" ;;
-		*) echo -e "[?] UNKNOWN: ${MESSAGE}" ;;
+		info) echo -e "[${BGreen}+${NC}] INFO: ${MESSAGE}" ;;
```

Comment:
Fix variable names

Reply:
We use bolded variants in `pacstall`, so I copied that over to update it here, for consistency.

https://github.com/pacstall/pacstall/blob/5ada91da49a5bfec379041cf8bb8ca7eb834757b/pacstall#L122-L127

Reply:
Yes, but those aren't the names defined in this script:
![image](https://user-images.githubusercontent.com/65723952/177200656-c4d7c9df-94bc-441d-92e5-1870a1f4c692.png)